README.md

```diff
@@ -49,25 +49,12 @@ React native is based off of Node.js so we need to [install that first.](https:/
 so for Fedora: "sudo yum install nodejs npm" etc.
 
 To install react native we want to cd back to our project folder.
-First I want you to check the package.json and make sure that it reads under dependencies:
-
-    "react": "^15.3.2",
-    "react-native": "^0.40",
 
 If not then change the numbers to reflect the proper versions.
 
 Now run the following command:
 
-    sudo npm install
-    sudo npm install -g react-native-cli
-
-If this does not work when trying to run later then try the following commands:
-    sudo npm install react-native
-    sudo npm install react
-
-In the [react native docs](https://facebook.github.io/react-native/docs/getting-started.html ) they say watchman is optional,
-THEY LIE! YOU NEED IT! So install it using:
```

Comment:
I suppose this was confusion from failures in which the error references `watchman` but that is not generally the issue. Should read the referenced issue on the `react-native` if that was the case.